extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -442,7 +445,11 @@ export class SchemaCompareOptionsDialog {
 		this.SetDeploymentOptions();
 		this.SetObjectTypeOptions();
 		if (this.optionsChanged) {
-			vscode.window.showInformationMessage(localize('schemaCompareOptions.recompareMessage', 'Options have changed. Press Compare to see the comparison.'));
+			vscode.window.showWarningMessage(SchemaCompareOptionsDialog.OptionsChangedMessage, SchemaCompareOptionsDialog.CompareButtonText).then(async (result) => {
```

Comment:
I think it make more sense to make this a modal dialog, otherwise user would be working on outdated compare results without knowing that. Udeesha has posted a comment on how to make it modal.

Reply:
I confirmed with Andrew to stick with the toast notification since the modal is blocking in the experience and they can decide to compare another time

Reply:
There should be 2 options for the user, Yes, No in the dialog. the user can choose No if they want to do the compare later. by doing this we are sure that the user is making an informed decision.

Reply:
Ok I updated it to have a yes and no button 
![image](https://user-images.githubusercontent.com/31145923/58518381-da8e1200-8163-11e9-8884-56ecb09b1d00.png)